atm.js

```diff
@@ -1,21 +1,63 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var checkingBalance_div = document.querySelector("div#checking_balance");
+var savingsBalance_div = document.querySelector("div#savings_balance");
+checkingBalance = 0;
+savingsBalance = 0;
+console.log(checkingBalance);
```

Comment:
I think instantiating two values of 0 is fine for this exercise. But eventually will figure out how to persist data in our applications and the balance wont always start out this way. I think better might be to access the balance through the DOM itself.